resources/analysisTools/indelCallingWorkflow/checkSampleSwap_TiN.pl

```diff
@@ -327,10 +389,38 @@
 
 `perl \${TOOL_NEW_COLS_TO_VCF} -vcfFile=$snvsGT_germlineRare_oVCF --newColFile=$snvsGT_germlineRare_oVCF.forAnnovar.temp --newColHeader=ANNOVAR_FUNCTION,GENE,EXONIC_CLASSIFICATION,ANNOVAR_TRANSCRIPTS --reportColumns=3,4,5,6 --bChrPosEnd=0,1,2 > $snvsGT_germlineRare_oVCF_annovar`;
 
+## Separating rare germline and rescued somatic variants
+open(RG, ">$snvsGT_germlineRare_oVCF_annovar_rG") || die "$snvsGT_germlineRare_oVCF_annovar_rG can't be open for writing. $!";
+open(SR, ">$snvsGT_germlineRare_oVCF_annovar_sR") || die "$snvsGT_germlineRare_oVCF_annovar_sR can't be open for writing. $!";
+open(GRA, "<$snvsGT_germlineRare_oVCF_annovar") || die "can't open $snvsGT_germlineRare_oVCF_annovar $!";
+
+while(<GRA>){
+  chomp;
+  if($_=~/^#/) {
+    print RG "$_\n";
+    print SR "$_\n";
+  }
+  elsif($_=~/Germline|SomaticControlRare/){
```

Comment:
Are you sure you don't want to match full length? I.e. `$row =~ /^(?:Germline|SomaticControlRare)$/`. Just in case there will by "XGermline", "GermlineY" or similar at some point in the future.

Reply:
This should be fine, these are controlled variables in the file.